File: TransCoder_tokenized_test_set_functions/java/CHECK_TWO_GIVEN_SETS_DISJOINT_1.java
boolean aredisjoint ( int set1 [ ], int set2 [ ] ) {
  int i = 0, j = 0;
  Arrays . sort ( set1 );
  Arrays . sort ( set2 );
  while ( i < set1 . length && j < set2 . length ) {
    if ( set1 [ i ] < set2 [ j ] ) i ++;
    else if ( set1 [ i ] > set2 [ j ] ) j ++;
    else return false;
  }
  return true;
}

File: TransCoder_tokenized_test_set_functions/cpp/CHECK_TWO_GIVEN_SETS_DISJOINT_1.cpp
bool areDisjoint ( int set1 [ ], int set2 [ ], int m, int n ) {
  sort ( set1, set1 + m );
  sort ( set2, set2 + n );
  int i = 0, j = 0;
  while ( i < m && j < n ) {
    if ( set1 [ i ] < set2 [ j ] ) i ++;
    else if ( set2 [ j ] < set1 [ i ] ) j ++;
    else return false;
  }
  return true;
}